src/components/form/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { ThemeUICSSObject } from 'theme-ui';
+
+const FormModalWrapper: ThemeUICSSObject = {
+  display: 'flex',
```

Comment:
Super nitpick, but we try to alphabetize the CSS :-/

Reply:
ill clean up any touched `styles` files here with the alphabetizing...will raise a separate PR for the other ones.